PlayMaker/PlayMaker/Resources/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Why it is added? And why it is added here?

Reply:
It was missing one to wrap all settings in two brackets.

Reply:
I removed this launchsettings json since it was not created on purpose by mistake